crates/matrix-sdk/Cargo.toml

```diff
@@ -16,20 +16,10 @@ features = ["docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = [
-    "e2e-encryption",
-    "automatic-room-key-forwarding",
-    "sqlite",
-    "native-tls",
-]
+default = ["e2e-encryption", "automatic-room-key-forwarding", "sqlite", "native-tls"]
 testing = []
 
-e2e-encryption = [
-    "matrix-sdk-base/e2e-encryption",
-    "matrix-sdk-base/message-ids",
-    "matrix-sdk-sqlite?/crypto-store",        # activate crypto-store on sqlite if given
-    "matrix-sdk-indexeddb?/e2e-encryption",   # activate on indexeddb if given
-]
+e2e-encryption = ["matrix-sdk-base/e2e-encryption", "matrix-sdk-base/message-ids", "matrix-sdk-sqlite?/crypto-store", "matrix-sdk-indexeddb?/e2e-encryption"]
```

Comment:
I agree with the other two, but this one seems to be quite long for it to be on a single line.

Reply:
This was not an intentional change. Not idea what happened there..